manifests/n/Neurodesk/NeurodeskApp/1.0.0/Neurodesk.NeurodeskApp.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Neurodesk.NeurodeskApp
+PackageVersion: 1.0.0
+Installers:
+- InstallerUrl: https://github.com/NeuroDesk/neurodesk-app/releases/latest/download/NeurodeskApp-Setup-Windows.exe

```

```suggestion
- InstallerUrl: https://github.com/NeuroDesk/neurodesk-app/releases/download/v1.0.0/NeurodeskApp-Setup-Windows.exe
```
Best to use versioned URL since @wingetbot will likely not automatically update this package due to the amount of packages in the community repository.